core/lib/prover_interface/tests/job_serialization.rs

```diff
@@ -76,3 +78,105 @@ async fn test_final_proof_deserialization() {
+            minor: ProtocolVersionId::Version25,
+            patch: 10.into(),
+        },
+    }));
+    let encoded_obj = serde_json::to_string(&proof).unwrap();
+    let encoded_json = r#"{
+        "Proof": {
+            "aggregation_result_coords": [
+                [
+                    0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+                ],
+                [
+                    0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+                ],
+                [
+                    0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+                ],
+                [
+                    0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+                ]
+            ],
+            "scheduler_proof": {
+                "n": 0,
+                "inputs": [],
+                "state_polys_commitments": [],
+                "witness_polys_commitments": [],
+                "copy_permutation_grand_product_commitment": {
+                    "x": [ 0, 0, 0, 0 ],
+                    "y": [ 1, 0, 0, 0 ],
+                    "infinity": true
+                },
+                "lookup_s_poly_commitment": null,
+                "lookup_grand_product_commitment": null,
+                "quotient_poly_parts_commitments": [],
+                "state_polys_openings_at_z": [],
+                "state_polys_openings_at_dilations": [],
+                "witness_polys_openings_at_z": [],
+                "witness_polys_openings_at_dilations": [],
+                "gate_setup_openings_at_z": [],
+                "gate_selectors_openings_at_z": [],
+                "copy_permutation_polys_openings_at_z": [],
+                "copy_permutation_grand_product_opening_at_z_omega": [ 0, 0, 0, 0 ],
+                "lookup_s_poly_opening_at_z_omega": null,
+                "lookup_grand_product_opening_at_z_omega": null,
+                "lookup_t_poly_opening_at_z": null,
+                "lookup_t_poly_opening_at_z_omega": null,
+                "lookup_selector_poly_opening_at_z": null,
+                "lookup_table_type_poly_opening_at_z": null,
+                "quotient_poly_opening_at_z": [ 0, 0, 0, 0 ],
+                "linearization_poly_opening_at_z": [ 0, 0, 0, 0 ],
+                "opening_proof_at_z": {
+                    "x": [ 0, 0, 0, 0 ],
+                    "y": [ 1, 0, 0, 0 ],
+                    "infinity": true
+                },
+                "opening_proof_at_z_omega": {
+                    "x": [ 0, 0, 0, 0 ],
+                    "y": [ 1, 0, 0, 0 ],
+                    "infinity": true
+                }
+            },
+            "protocol_version": "0.25.10"
+        }
+    }"#;
+    let decoded_obj: SubmitProofRequest = serde_json::from_str(&encoded_obj).unwrap();
+    println!("{}", encoded_obj);
```

Comment:
We should not use `println` in tests.

Reply:
[Fixed](https://github.com/matter-labs/zksync-era/pull/1993/commits/4103ae01a0237930818b2cc88b1164613bf3f611) ✅